frontend/src/main/web/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+This node script will loop through all sibling directories and invoke npm command on each one of them.
+To use this script, run: node index.js install or node index.js run build
```

Comment:
I thought we were going to go with something simpler, but where we explicitly list the sub-modules to build, instead of walking the directories using a script.  Possibly using a Makefile.